waspc/examples/todoApp/src/pages/ProfilePage.tsx

```diff
@@ -41,15 +45,13 @@ export const ProfilePage = ({ user }: { user: User }) => {
   ))
   const connectionIcon = isConnected ? '🟢' : '🔴'
 
-  const providerData = getProviderData(user)
-
   return (
     <>
       <h2>Profile page</h2>
       <div>
         Hello <strong>{getName(user)}</strong>! Your status is{' '}
         <strong>
-          {providerData && providerData.isEmailVerified
+          {user.identities.email && user.identities.email.data.isEmailVerified
```

Comment:
I commented somewher else I remember, I think `user.identities.email` is a bit misleading because it reads as if that is email, but it is not.
I wonder if `user.identities.emailAndPass` would be nicer therefore, I think it would. But, I guess that would need data migrations? Maybe not? Or maybe it wouldn't be consistent with the rest of the naming we use, if we use just "email" everywhere for "email and pass"?

Reply:
We use `email` everywhere right now. We were pretty consistent with that. 

For all other providers we followed the same logic, put the provider name as the key in the `identities` object.

Reply:
@infomiho it is not about the position of `email`, it makes that it is in `identities`, it is that it would be clearer what it is if it was called `emailAndPass`.
Same probably stands for `username`, if there is also `identities.username`.

Maybe it is worth changing here, even if we `email` in imports? In imports i also guess context is a bit different, and they are more autmoatized, and nobody reads them directly.

Reply:
Aha! Oh, ok if that is so, then indeed it would be nicest if we went with `emailAndPassword` everywhere. Maybe better to do that now than later.
But we don't have to do it in this PR if you think it is too much, we can do it in another PR? But let's try to do it soon after this one?
What is your feeling about this, does it make sense to do this change? Sounds nice to me, like we have nice / consistent naming, but I think you are a bit better suited to judge if it is worthy the effort.

Reply:
Okay, I've created and issue and I'll make sure to tackle this soon after we merge this one: https://github.com/wasp-lang/wasp/issues/1947

I'd go for renaming stuff completely: `email` -> `emailAndPassword` and also making sure we use `usernameAndPassword` everywhere.